packages/react/src/publication/useBookmarkToggle.ts

```diff
@@ -0,0 +1,84 @@
+import { AnyPublication, ProfileOwnedByMe, isMirrorPublication } from '@lens-protocol/api-bindings';
+import { invariant, success } from '@lens-protocol/shared-kernel';
+
+import { Operation, useOperation } from '../helpers/operations';
+import { useBookmarkToggleController } from './adapters/useBookmarkToggleController';
+
+export type UseBookmarkToggleArgs = {
+  profile: ProfileOwnedByMe;
+  publication: AnyPublication;
+};
+
+export type BookmarkOperation = Operation<void>;
+
+/**
+ * `useBookmarkToggle` hook let the user save or remove a publication from their bookmarks.
+ *
+ * You MUST be authenticated via {@link useWalletLogin} to use this hook.
+ *
+ * You can use the {@link Post.bookmarked} (or  {@link Comment.bookmarked}) property to determine
```

Comment:
I saw while trying to generate typedoc yesterday that these links to Fragment props don't work, and typedoc complains about them

Reply:
Yeah, it's part of the syntax but struggled to make it work: https://typedoc.org/tags/link/

Or something I don't fully understand.